examples/kuadrant/.gitignore

```diff
@@ -1,2 +1,3 @@
 bin
 topology.dot
+/kuadrant
```

Comment:
?

Reply:
If you run `go build` over using the make target you get the binary which I assume we never want included. I would be happy to remove. The leading / was added by goland, that is possibly not required, now that I look at it again.

Reply:
The example Makefile don't have a `build` target. In case we add one, we can make it PHONY and it won't spit out the binary.

Reply:
I know there is no Makefile `build` target. I ran `go build` directly. This was partly habit but also setting a run target in Goland so I could attach a debugger process. I can remove it there is no problem.